dataservice/api/sample/models.py

```diff
@@ -8,16 +9,24 @@ class Sample(db.Model, Base):
     :param _id: Unique id assigned by RDBMS
     :param kf_id: Unique id given by the Kid's First DCC
     :param external_id: Name given to sample by contributor
-    :param composition : composition of the sample
-    :param tissue_type: Either Normal or Tumor
-    :param anatomical_site : location of the collected sample
+    :param composition : The cellular composition of the sample.
+    :param tissue_type: description of the kind of tissue collected
+           with respect to disease status or proximity to tumor tissue
+    :param anatomical_site : The name of the primary disease site of the
+           submitted tumor sample
     :param age_at_event_days: age of participant when sample was collected
+    :param tumor_descriptor: The kind of disease present in the tumor
+           specimen as related to a specific timepoint
     """
     __tablename__ = "sample"
     external_id = db.Column(db.Text())
     tissue_type = db.Column(db.Text())
     composition = db.Column(db.Text())
     anatomical_site = db.Column(db.Text())
     age_at_event_days = db.Column(db.Integer())
-    participant_id = db.Column(db.Integer, db.ForeignKey('participant.kf_id'),
+    tumor_descriptor = db.Column(db.Text())
+    aliquots = db.relationship(Aliquot, backref='samples',
```

Comment:
@parimalak It looks like you made the changes to the models already. The way you have the relationships configured for Sample and Aliquot are correct now. This will achieve the one-to-many relationship going from Sample to Aliquots and also when you delete a Sample, it will delete all associated Aliquots. When you delete all the Aliquots for a given Sample, the Sample will still exist which is what we want. 👍